portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,29 @@
-#content {
+/* Create flex box */
+
+#container {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```

Comment:
Please make the css properrties alphabetically ordered here and below. So,
display:
flex-direction:
justify-content:
etc